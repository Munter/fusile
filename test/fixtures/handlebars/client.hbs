<h1>Under the {{ body_of_water }}</h1>