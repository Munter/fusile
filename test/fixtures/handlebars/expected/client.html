Handlebars.template({"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
  var helper, functionType="function", helperMissing=helpers.helperMissing, escapeExpression=this.escapeExpression;
  return "<h1>Under the "
    + escapeExpression(((helper = (helper = helpers.body_of_water || (depth0 != null ? depth0.body_of_water : depth0)) != null ? helper : helperMissing),(typeof helper === functionType ? helper.call(depth0, {"name":"body_of_water","hash":{},"data":data}) : helper)))
    + "</h1>";
},"useData":true});